mikelpardo/mobile-sales
# Add search mobile functionality

**Description**

Add search functionality in home view

**How to test**

1. Go to: http://localhost:3000/
2. Use search input and check that its filter correctly by mobile model or brand
